Trappist/src/Promact.Trappist.DomainModel/ApplicationClasses/Test/TestAC.cs

```diff
@@ -0,0 +1,14 @@
+﻿
+using System.Collections.Generic;
+
+namespace Promact.Trappist.DomainModel.ApplicationClasses.Test
+{
+    public class TestAC
+    {
+        public int Id { get; set; }
+       
+        public string TestName { get; set; }
+       
+        public List<CategoryAC> CategoryACList { get; set; }
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename property 'CategoryACList' to match camel case naming rules, consider using 'CategoryAcList'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS100)